.drone.yml

```diff
@@ -0,0 +1,438 @@
+      instance:
+        - drone-publish.rancher.io
+      status:
+        - failure
+
+volumes:
+  - name: docker_pipe
+    host:
+      path: \\\\.\\pipe\\docker_engine
+
+trigger:
+  event:
+    exclude:
+      - promote
+---
+kind: pipeline
+name: windows-2004
+
+platform:
+  os: windows
+  arch: amd64
+  version: 2004
+
+# remove this and use upstream images when https://github.com/drone/drone-git/pull/25 is merged
+clone:
+  disable: true
+
+steps:
+  - name: clone
+    image: luthermonson/drone-git:windows-2004-amd64
+  - name: build
+    pull: always
+    image: mcr.microsoft.com/windows/servercore:2004
+    commands:
+      - docker build -f Dockerfile.windows
+    volumes:
+      - name: docker_pipe
+        path: \\\\.\\pipe\\docker_engine
+    when:
+      event:
+        - push
+        - pull_request
+        - tag
+
+  - name: docker-publish
+    image: luthermonson/drone-docker:2004
+    settings:
+      build_args:
+        - WINDOWS_VERSION=2004
+        - ARCH=amd64
+        - VERSION=${DRONE_TAG}
+      # context:
+      # FIXME: keeping this here for review in case we need it
+      custom_dns: 1.1.1.1
+      dockerfile: Dockerfile.windows
+      username:
+        from_secret: docker_username
+      password:
+        from_secret: docker_password
+      repo: rancher/fluent-bit
+      tag: ${DRONE_TAG}-windows-2004
+    volumes:
+      - name: docker_pipe
+        path: \\\\.\\pipe\\docker_engine
+    when:
+      event:
+        - tag
+      ref:
+        - refs/heads/master
+        - refs/tags/*
+
+  - name: slack_notify
+    image: plugins/slack
+    settings:
+      template: "Build {{build.link}} failed to publish an image/artifact.\n"
+      username: Drone_Publish
+      webhook:
+        from_secret: slack_webhook
+    when:
+      event:
+        exclude:
+          - pull_request
+      instance:
+        - drone-publish.rancher.io
+      status:
+        - failure
+
+volumes:
+  - name: docker_pipe
+    host:
+      path: \\\\.\\pipe\\docker_engine
+
+trigger:
+  event:
+    exclude:
+      - promote
+---
+kind: pipeline
+name: windows-20H2
+
+platform:
+  os: windows
+  arch: amd64
+  version: 20H2
+
+# remove this and use upstream images when https://github.com/drone/drone-git/pull/25 is merged
+clone:
+  disable: true
+
+steps:
+  - name: clone
+    image: luthermonson/drone-git:windows-20H2-amd64
+  - name: build
+    pull: always
+    image: mcr.microsoft.com/windows/servercore:20H2
+    commands:
+      - docker build -f Dockerfile.windows
+    volumes:
+      - name: docker_pipe
+        path: \\\\.\\pipe\\docker_engine
+    when:
+      event:
+        - push
+        - pull_request
+        - tag
+
+  - name: docker-publish
+    image: luthermonson/drone-docker:20H2
+    settings:
+      build_args:
+        - WINDOWS_VERSION=20H2
+        - ARCH=amd64
+        - VERSION=${DRONE_TAG}
+      # context:
+      # FIXME: keeping this here for review in case we need it
+      custom_dns: 1.1.1.1
```

Comment:
why do we use a custom dns?

Reply:
This is a Windows-specific DNS setting that we use across multiple Drone configs: https://github.com/rancher/dapper/blob/master/.drone.yml#L106

I honestly do not have an explanation as to _why_ we do this.
CC: @luthermonson